src/PointwiseFunctions/AnalyticData/BnsInitialData/SpecData.cpp

```diff
@@ -0,0 +1,118 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/BnsInitialData/SpecData.hpp"
+
+#include <Exporter.hpp>  // The SpEC Exporter
+#include <memory>
+#include <pup.h>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/EagerMath/RaiseOrLowerIndex.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "IO/External/InterpolateFromSpec.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/Hydro/SpecificEnthalpy.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/System/ParallelInfo.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace BnsInitialData::AnalyticData {
+
+template <size_t ThermodynamicDim>
+SpecData<ThermodynamicDim>::SpecData(
+    std::string data_directory,
+    std::unique_ptr<equation_of_state_type> equation_of_state,
+    const double density_cutoff, const double orbital_angular_velocity,
+    const double euler_enthalpy_constant)
+    : data_directory_(std::move(data_directory)),
+      equation_of_state_(std::move(equation_of_state)),
+      density_cutoff_(density_cutoff),
+      orbital_angular_velocity_(orbital_angular_velocity),
+      euler_enthalpy_constant_(euler_enthalpy_constant),
+      spec_exporter_(std::make_unique<spec::Exporter>(1, data_directory_,
```

Comment:
Hmm @nilsvu, it seems that if this is set to 1 then my test runs, but the executable gives some weird runtime error and if it's set to `sys::props_on_node(sys::my_node())` then the test segfaults but the executable runs no problem